src/parser/permit-generation-module.ts

```diff
@@ -175,14 +179,15 @@ export class PermitGenerationModule implements Module {
     let permitFeeAmountDecimal = new Decimal(0);
     for (const [_, rewardResult] of Object.entries(result)) {
       // accumulate total permit fee amount
-      const totalAfterFee = +(new Decimal(rewardResult.total).mul(feeRateDecimal));
+      const totalAfterFee = +new Decimal(rewardResult.total).mul(feeRateDecimal);
```

Comment:
```suggestion
      const totalAfterFee = Number(new Decimal(rewardResult.total).mul(feeRateDecimal));
```

Seems that you discarded all my review changes from the other day. These should be wrapped with `Number();` instead of prefixing with `+` be sure to also do for the others.

Reply:
It is possible that I added these aforementioned changes on another pull I'm not entirely sure, but I remember adding this to my review.

Reply:
I think a `toNumber()` should make more sense here, will have a look.

Reply:
I made these changes in #67